benchmarks/bench_evolve.py

```diff
@@ -30,10 +32,16 @@ def benchmark_write_operator_card_from_file(tmp_path, test_files, test_configs):
     myopcard = yaml.safe_load(target_path.read_text(encoding="utf-8"))
     # Check if it contains all the information for eko
     assert np.allclose(myopcard["rotations"]["xgrid"], x_grid)
-    assert np.allclose(myopcard["rotations"]["targetgrid"], x_grid)
-    assert np.allclose(myopcard["rotations"]["inputgrid"], x_grid)
-    assert np.allclose(myopcard["rotations"]["inputpids"], pineko.evolve.DEFAULT_PIDS)
-    assert np.allclose(myopcard["rotations"]["targetpids"], pineko.evolve.DEFAULT_PIDS)
+    # Checking that inputpids and targetpids have been correctly removed
+    with pytest.raises(KeyError):
+        myopcard["rotations"]["inputpids"]
```

Comment:
I think the `with` breaks here and the rest is never executed (so you need to repeat it - consider a loop ;-) )

Reply:
Oh you are right :)